neurots/generate/algorithms/basicgrower.py

```diff
@@ -22,6 +22,8 @@
 from neurots.generate.algorithms.abstractgrower import AbstractAlgo
 from neurots.generate.algorithms.common import bif_methods
 from neurots.generate.algorithms.common import section_data
+from neurots.morphmath import rotation
+from neurots.utils import PIA_DIRECTION
```

Comment:
Maybe pia direction is too cortical, what about growth_direction?

Reply:
principal_direction?

Reply:
sure! ah, this was introduced in the trunk modification?